privacy_evaluator/models/tf/cnn.py

```diff
@@ -73,13 +87,14 @@ def __init__(
     def call(self, x):
         out = self.conv1(x)
         out = self.conv2(out)
+        out = self.conv3(out)
         out = self.fc(out)
         return out
 
 
 class ConvNetCIFAR10(keras.Model):
     def __init__(
-        self, num_classes: int = 2, num_channels: Tuple[int, ...] = (3, 64, 128)
+        self, num_classes: int = 2, num_channels: Tuple[int, ...] = (3, 16, 32, 64)
     ):
```

Comment:
Same question as above, is there a specific reason for these default value of num_classes?